### PR TITLE
Codeowners for resource manager path

### PR DESCRIPTION
**Pull request by anamikapan**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by ronniegeraghty — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -927,10 +927,10 @@
 # AzureSdkOwners:                                                  @ArthurMa1978
 
 # PRLabel: %ARM
-/sdk/resources/Azure.ResourceManager.*/                            @Azure/arm-sdk-owners
+/sdk/resources/Azure.ResourceManager.*/                            @arm-sdk-owners
```
Comment (jsquire) [inline .github/CODEOWNERS:930]: This is not a known GitHub user.    If this is a team, the format is incorrect.  The organization under which the team was created needs to be used as a prefix, as  it was in the previous iteration.

**Review by jsquire — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -927,10 +927,10 @@
 # AzureSdkOwners:                                                  @ArthurMa1978
 
 # PRLabel: %ARM
-/sdk/resources/Azure.ResourceManager.*/                            @anamikapan
+/sdk/resources/Azure.ResourceManager.*/                            @arm-sdk-owners
```
Comment (weshaggard) [inline .github/CODEOWNERS:930]:
```suggestion
/sdk/resources/Azure.ResourceManager.*/                            @Azure/arm-sdk-owners
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -927,10 +927,10 @@
...
 
 # ServiceLabel: %ARM %Mgmt
-# ServiceOwners:                                                   @anamikapan
+# ServiceOwners:                                                   @arm-sdk-owners
```
Comment (weshaggard) [inline .github/CODEOWNERS:933]:
```suggestion
# ServiceOwners:                                                   @Azure/arm-sdk-owners
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (weshaggard) [reply .github/CODEOWNERS:930]: I just added arm-sdk-owners as a child of azure-sdk-write but a maintainer of that team will have to except that change.

Comment (anamikapan) [reply .github/CODEOWNERS]: @weshaggard can I know by when would we expect to know the approval from maintainer?

Comment (weshaggard) [reply .github/CODEOWNERS]: You will have to ask a maintainer of that team to accept the parent update. @cemheren @majastrz or @anthony-c-martin to accept the update to re-parent arm-sdk-owners to azure-sdk-write.

**Review by majastrz — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (majastrz) [reply .github/CODEOWNERS]: I just approved the change.

Comment (majastrz) [reply .github/CODEOWNERS]: Never mind. I only approved the PR. I haven't seen the request to reparent the group.

Comment (weshaggard) [reply .github/CODEOWNERS]: @majastrz if you go to the team https://github.com/orgs/Azure/teams/arm-sdk-owners and look at the settings it should show a request to parent to azure-sdk-write.

Comment (majastrz) [reply .github/CODEOWNERS]: Finally found the approval email and approved it. The team should be parented correctly now:
image

**Review by jsquire — APPROVED** [review]

**Pull request merged**